src/js/components/Button/__tests__/__snapshots__/Button-kind-test.js.snap

```diff
@@ -311,6 +311,98 @@ exports[`Button kind custom theme secondary 1`] = `
 </div>
 `;
 
+exports[`Button kind disabled with hoverIndicator should not hover 1`] = `
+.c1 {
+  display: inline-block;
+  box-sizing: border-box;
+  cursor: pointer;
+  font: inherit;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  margin: 0;
+  background: transparent;
```

Comment:
Just to confirm if this test had existed, without the fix for grommet, this would have received a background color and now it's demonstrating that no background is applied on hover?

Reply:
Good question, the test didn't exist before, I added the test locally before applying the fix, and after applying the fix I've verified that the background color was removed as it should.

Reply:
Great. Thanks!